runtime/oti/j9nonbuilder.h

```diff
@@ -34,6 +34,9 @@
 #include "j9consts.h"
 #include "jvmti.h"
 #include "j9javaaccessflags.h"
+#if JAVA_SPEC_VERSION >= 16
+#include <setjmp.h>
```

Comment:
I would be tempted to only include this file in the setjmp wrapper file (and declare the J9VMThread variable as `void*`.

Reply:
Agreed and updated the code against the suggestion above.